.size-limit

```diff
@@ -1,24 +1,24 @@
 [
   {
     "path": "dist/index.cjs",
-    "limit": "2.2 KB",
+    "limit": "2.3 KB",
     "webpack": false,
     "running": false
   },
   {
     "path": "dist/index.cjs",
-    "limit": "700 B",
-    "import": "{ Chart }"
```

Comment:
Now we don't have common chart component?

Reply:
@thabarbados
1) What is VueCharts for? May be better to remove it?
2) Can we made `<Chart type="...">` component?

Reply:
@dangreen 

1. VueChart is is Object with all kinds of charts. With it, you can use VueCharts.Line instead of Line
2. No, we can't do that. We can make components <Bar />, <Line /> etc.

Reply:
@thabarbados looks like we can remove it, because we can do

```
import * as VueCharts from 'vue-chartjs'
```

Reply:
@dangreen 
Ok. I will remove it.

Which module when to use for size-limit test instead of `"import": "{ Chart }"` ? "import": "{ Bar }"?

Reply:
Bar

Reply:
fixed